src/main/scripts/chocolatey-package.bash

```diff
@@ -152,12 +159,20 @@ function main() {
         return 1
     fi
 
-    if ! update_resources $project_version target/rug-cli-${project_version}-bin.zip; then
+    # nuget doesn't support semver such as 1.0.0-m.2
+    # the .2 trail is not supported, so we create a nuget package without the dot
+    # see https://github.com/atomist/rug-cli/issues/189
+    local valid_nuget_project_version=$project_version
+    if [[ $project_version =~ ^[0-9]+\.[0-9]+\.[0-9]+(-(m|rc)\.[0-9]+)$ ]]; then
+        valid_nuget_project_version=$(echo $project_version | sed -rn 's/([0-9]+\.[0-9]+\.[0-9]+)(-m|-rc)\.([0-9]+)/\1\2\3/p')
```

Comment:
Updated as per your snippet. Tried on Ubuntu and that worked well there too. Thanks!